x-pack/plugins/ml/public/application/components/annotations/annotations_service/annotation_service.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { AnnotationUpdatesService } from '../../../services/annotations_service';
+
+interface MlAnnotationWrapperProps {
+  children: (annotationUpdatesService: AnnotationUpdatesService) => React.ReactElement;
+}
+
+export const MlAnnotationComponent: FC<MlAnnotationWrapperProps> = ({ children }) => {
```

Comment:
Please remove this component and use react hooks instead

Reply:
Updated it to using a react context instead here  https://github.com/elastic/kibana/pull/83216/commits/81e20c46cea66e0ca707173eb0c31ef3365f0478